src/strategies/index.ts

```diff
@@ -589,7 +591,9 @@ const strategies = {
   'gno-vote-weight': gno,
   'uma-voting': umaVoting,
   'masterchef-pool-balance-no-rewarddebt': masterchefPoolBalanceNoRewarddebt,
-  'proof-of-humanity': proofOfHumanity
+  'proof-of-humanity': proofOfHumanity,
+  'fantomstarter-staked-nft': fantomStarterStakedNFT,
+  'fantomstarter-owned-nft': fantomStarterOwnedNFT
```

Comment:
Hi @Igoranze Please create separate PRs for these two strategies, it will be easy to review/integrate/refer